ledger/participant-integration-api/src/test/suite/scala/platform/indexer/RecoveringIndexerSpec.scala

```diff
@@ -102,7 +102,7 @@ class RecoveringIndexerSpec
       }
     }
 
-    "wait until the subscription completes" in newLoggingContext { implicit logCtx =>
+    "wait until the subscription completes" in newLoggingContext { implicit loggingContext =>
```

Comment:
See if it makes sense to use `LoggingContext.ForTesting` here and remove a level of nesting.